src/ui/BookmarksComboBoxWidget.cpp

```diff
@@ -41,6 +41,7 @@ BookmarksComboBoxWidget::BookmarksComboBoxWidget(QWidget *parent) : ComboBoxWidg
 void BookmarksComboBoxWidget::createFolder()
 {
 	const QString title(QInputDialog::getText(this, tr("Folder Name"), tr("Select name of new folder:")));
+	BookmarksModel::Bookmark *prevFolder = getCurrentFolder();
```

Comment:
We avoid this kind of initialization syntax, it should be:
```
BookmarksModel::Bookmark *previousFolder(getCurrentFolder());
```
And I think that it could be fixed in `BookmarksComboBoxWidget::setCurrentFolder()` instead, without adding condition (always expand).